README.md

```diff
@@ -85,6 +88,42 @@ steps:
       approve-only: true
 ```
 
+### Trigger action from workflow_dispatch event
+
+If you need to trigger this action manually, you can use the [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) event. A use case might be that your CI runs on a seperate provider, so you would like to run this action as a result of a successful CI run.
+
+When using the `workflow_dispatch` approach, you will need to send the PR number as part of the input for this action:
+
+```yml
+name: automerge
+
+on: 
+  workflow_dispatch:
+    inputs:
+      pr:
+        required: true
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: fastify/github-action-merge-dependabot@v2.2.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          pr: ${{ github.event.inputs.pr }}
```

Comment:
the action parameter name is wrong, it should be `pr-number`. I'm not actually 100% sure this is a valid identifier when used as `github.event.inputs.pr-number` though

```suggestion
          pr-number: ${{ github.event.inputs.pr-number }}
```